common/build.gradle.kts

```diff
@@ -28,6 +28,7 @@ plugins {
 
 dependencies {
   implementation(project(":api"))
+  implementation(project(":catalogs:catalog-common"))
```

Comment:
The dependency here looks strange if you only want to define two properties in catalog-common, it is easy to introduce the cyclic dependency.

Reply:
catalog-common is mainly used to define some properties used by catalog, client, connectors, I think it's ok to add the dependence here.

Reply:
No, it's weird to make common module rely on catalog-common module, which have different propose and easy to introduce cyclic dependency, please change it.

Reply:
ok